iceoryx_hoofs/testing/testing_logger.cpp

```diff
@@ -112,15 +119,54 @@ std::vector<std::string> TestingLogger::getLogMessages() noexcept
     return logger.m_loggerData->buffer;
 }
 
+#if !defined(_WIN32)
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables) global variable is required as jmp target
+jmp_buf exitJmpBuffer;
+
+static void sigsegvHandler(int sig, siginfo_t*, void*)
```

Comment:
Since this is not only a handler for SIGSEGV, shall we rename it to something like `sigHandler`?